README.md

```
@@ -1,61 +1,409 @@
-# Skyra.PW [![Discord](https://discord.com/api/guilds/254360814063058944/embed.png)](https://join.skyra.pw)
+# WolfStar.rocks
 
-[![Docker Pulls](https://img.shields.io/docker/pulls/skyrabot/skyra.pw?logo=docker&logoColor=white)](https://hub.docker.com/r/skyrabot/skyra.pw)
+<div align="center"><a name="readme-top"></a>
 
-## Set-Up
+<img src="https://github.com/WolfStarBot.png" width="15%" alt="WolfStar Logo">
 
-To run the dashboard locally, you'll need a local version of Skyra running.
+# WolfStar.rocks
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)